packages/adapter-cloudflare/index.js

```diff
@@ -239,3 +241,11 @@ function generate_headers(app_dir) {
 # === END AUTOGENERATED SVELTE IMMUTABLE HEADERS ===
 `.trimEnd();
 }
+
+/** @param {import('@sveltejs/kit').Builder} builder */
+function generate_redirects(builder) {
+	return Array.from(
+		builder.prerendered.redirects.entries(),
+		([path, redirect]) => `${path} ${redirect.location} ${redirect.status}`
+	).join('\n');
+}
```

Comment:
We should follow `generate_headers` in ensuring we:
1. start on a newline so that it doesn't conflict with existing _redirects rules that do not end with a new line.
2. if possible, add the opening and closing comments to denote these are autogenerated. I think cloudflare has no special syntax for this and it can be written in plain text and their system will just ignore it since they're not valid redirect rules.

Another nice thing would be to throw an error if it exceeds the _redirects limit (although they are quite generous so maybe this would never be an issue and we don't need to expend this extra effort)